src/types/big-serial.ts

```diff
@@ -24,7 +24,7 @@ export const types = {
 	},
 };
 
-export type Types = TypeUtils.TsTypes<bigint, number | bigint>;
+export type Types = TypeUtils.TsTypes<string, string | number | bigint>;
```

Comment:
Does this file not need a `fetchProcessing`? I would expect it to be essentially identical to `big-integer` outside of the db type including the auto incrementing stuff

Reply:
If you notice at the top of the file I'm exporting the `fetchProcessing` from `big-integer` for the reason you mentioned of them essentially being identical. Is there something we should be doing specifically for the auto incrementing? I checked `serial` for an example, but it doesn't have a `fetchProcessing` to reference.

Reply:
I just didn't see the export is all, I assumed it was just imports that were necessary and skipped over it